modules/kotest/build.gradle

```diff
@@ -0,0 +1,39 @@
+plugins {
+    id "org.jetbrains.kotlin.jvm" version "1.3.71"
+    id "org.jetbrains.kotlin.kapt" version "1.3.71"
+}
+
+ext {
+    kotestVersion = "4.0.3"
+}
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile project(':testcontainers')
+    api("io.kotest:kotest-runner-junit5-jvm:$kotestVersion")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    testImplementation("io.kotest:kotest-assertions-core-jvm:$kotestVersion")
+    testImplementation("io.mockk:mockk:1.9.3")
```

Comment:
not needed?

Reply:
Removed mockk dependency as now we have integration test only